Index: src/contacts-gtk.c
===================================================================
--- src/contacts-gtk.c	(revision 213)
+++ src/contacts-gtk.c	(working copy)
@@ -83,11 +83,12 @@
 		gtk_list_store_insert_with_values (list, &iter, 1, 0, data, -1);
 	}
 	GtkListStore *list;
+	GtkTreeIter dummy;
 
 	list = (GtkListStore*) gtk_combo_box_get_model (GTK_COMBO_BOX(groups_combobox));
 	if (!list) return;
 	gtk_list_store_clear (list);
-	gtk_list_store_insert_with_values (list, NULL, 0, 0, "All", -1);
+	gtk_list_store_insert_with_values (list, &dummy, 0, 0, "All", -1);
 	g_list_foreach (data->contacts_groups, add_to_list, list);
 
 	/* Select 'All' in the groups combobox if nothing selected */
