--- contacts-0.1/src/contacts-utils.c	2005-12-14 17:07:32.000000000 +0000
+++ /home/ross/Programming/cvs/contacts/trunk/src/contacts-utils.c	2006-10-17 21:23:57.000000000 +0100
@@ -357,8 +388,8 @@
 			switch (photo->type) {
 			case E_CONTACT_PHOTO_TYPE_INLINED :
 				gdk_pixbuf_loader_write (loader,
-					photo->inlined.data,
-					photo->inlined.length, NULL);
+					photo->data.inlined.data,
+					photo->data.inlined.length, NULL);
 				break;
 			case E_CONTACT_PHOTO_TYPE_URI :
 			default :
@@ -545,9 +576,9 @@
 				int *length;
 #if HAVE_PHOTO_TYPE
 				new_photo.type = E_CONTACT_PHOTO_TYPE_INLINED;
-				data = &new_photo.inlined.data;
-				length = &new_photo.inlined.length;
-				new_photo.inlined.mime_type = NULL;
+				data = &new_photo.data.inlined.data;
+				length = &new_photo.data.inlined.length;
+				new_photo.data.inlined.mime_type = NULL;
 #else
 				data = &new_photo.data;
 				length = &new_photo.length;
